lib/web/main.go

```
@@ -29,6 +30,13 @@ type Page struct {
 func (h *Handlers) GeneratePageData() *Page {
     title := "CI Dashboard"
 
+    fmt.Println("Generating Page Data.")
+
+    // Does file exist - otherwise return.
+
+    // if ()
+
+    utils.Lock()
     csvFile, _ := os.Open("result.csv")
```

a good few errors being ignored, i'd log them at least for your own debugging sanity

---

Think I'm checking all of these now.